src/ore/src/tracing/mod.rs

```diff
@@ -218,24 +253,14 @@ where
     let service_name = service_name.to_string();
 
     let config = config.into();
-    // See: https://no-color.org/
-    let no_color = std::env::var_os("NO_COLOR").unwrap_or_else(|| "".into()) != "";
-    let stderr_log_layer = fmt::layer()
-        .event_format(PrefixFormat {
-            inner: format(),
-            prefix: config.stderr_log.prefix,
-        })
-        .with_writer(io::stderr)
-        .with_ansi(!no_color && atty::is(atty::Stream::Stderr))
-        .with_filter(config.stderr_log.filter);
-
-    let (stderr_log_layer, stderr_reloader) = reload::Layer::new(stderr_log_layer);
+    let (stderr_log_filter, stderr_reloader) = reload::Layer::new(config.stderr_log.filter.clone());
     let stderr_callback = DynamicTargetsCallback {
         callback: Arc::new(move |targets| {
-            stderr_reloader.modify(|layer| *layer.filter_mut() = targets)?;
+            stderr_reloader.reload(targets)?;
```

Comment:
can we make line 323 look exactly like this? they are ostensibly the exact same thing so should look the same!